website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,133 @@
+﻿
+---
+title: "Intro to Karpenter Concepts"
+linkTitle: "Concepts"
+weight: 20
+resources:
+- src: "**fig1*.png"
+- src: "**fig2*.png"
+---
+
+Kubernetes is designed to schedule new pods onto existing cluster nodes. However, when existing nodes don't have available capacity, new Pods end up in a Pending state

```

Comment:
Let's reorient wording like "designed to" and just say what it does "The Kubernetes scheduler assigns incoming pods onto existing cluster nodes"